src/ceph_osd.cc

```diff
@@ -73,6 +73,10 @@ static void usage()
        << "  --osd-journal PATH\n"
        << "                    journal file or block device\n"
        << "  --mkfs            create a [new] data directory\n"
+       << "  --mkkey           generate a new secret key\n"
+       << "  --monmap          specify the path to the monitor map\n"
+       << "  --osd-uuid        specify the OSD's fsid\n"
+       << "  --keyring         specify a path to the osd keyring\n"
```

Comment:
please see @gregsfortytwo 's comment at http://tracker.ceph.com/issues/20057#note-8, except for `--mkkey`, all these options  are parsed by `md_config_t::parse_argv()`, which checks the args against **all** config_opts.  so, we can override all options by passing it to `ceph-osd`, like 
```
ceph-osd --osd-pool-default-pg-num
```

but shall we document them all also? i don't really think so.